### PR TITLE
Switched VertexAI models from preview

### PR DESCRIPTION
**Pull request by lkuligin**
Replace this entire comment with:
  - **Description:** VertexAI models are now GA, moved away from using preview ones from the SDK 
  - **Issue:** #13606

### CODE REVIEW

**Review by leo-gan — COMMENTED** [review]
lgtm

**Review by baskaryan — COMMENTED** [review]
should we bump min expected version in the import error?
```python
def raise_vertex_import_error(minimum_expected_version: str = "1.35.0") -> None:
    """Raise ImportError related to Vertex SDK being not available.

    Args:
        minimum_expected_version: The lowest expected version of the SDK.
    Raises:
        ImportError: an ImportError that mentions a required version of the SDK.
    """
    raise ImportError(
        "Could not import VertexAI. Please, install it with "
        f"pip install google-cloud-aiplatform>={minimum_expected_version}"
    )
```

**Comment by lkuligin** [comment]
@baskaryan added a few additional things:
- `candidate_count` behaviour was broken for codey models, fixed
- apparently, `get_num_tokens` works only for preview models, so introduced a `preview ` flag and an appropriate error message

File: libs/langchain/langchain/chat_models/vertexai.py [hunk 1]
```diff
@@ -132,17 +132,19 @@ def is_lc_serializable(self) -> bool:
     def validate_environment(cls, values: Dict) -> Dict:
         """Validate that the python package exists in environment."""
         cls._try_init_vertexai(values)
+        preview = values["preview"]
```
Comment (leo-gan) [inline libs/langchain/langchain/chat_models/vertexai.py:135]: My comment is not about this PR change so, ignore it if you like.
The `values` semantic is the `kwargs` semantic. I'd rename `values` to `kwargs`.

Comment (lkuligin) [reply libs/langchain/langchain/chat_models/vertexai.py:135]: noted, thanks, will change in the enxt iterations then!

**Comment by lkuligin** [comment]
@baskaryan @leo-gan I rebased on master, could you give it another look, please? Thanks!

**Review by leo-gan — APPROVED** [review]
lgtm with a small nit

File: libs/langchain/tests/integration_tests/llms/test_vertexai.py [hunk 2]
```diff
@@ -1,7 +1,5 @@
 """Test Vertex AI API wrapper.
-In order to run this test, you need to install VertexAI SDK (that is is the private
-preview)  and be whitelisted to list the models themselves:
-In order to run this test, you need to install VertexAI SDK 
+In order to run this test, you need to install VertexAI SDK:
 pip install google-cloud-aiplatform>=1.35.0
```
[the remark this answers is not in the document]
Comment (lkuligin) [reply libs/langchain/tests/integration_tests/llms/test_vertexai.py]: my bad, fixed!

**Review by leo-gan — APPROVED** [review]
lgtm

**Pull request merged**